src/officehours_ui/static/css/main.css

```diff
@@ -560,6 +559,22 @@ h3.alert-heading {
 	color: #cecbc9;
 }
 
+.table-responsive::-webkit-scrollbar {
+    -webkit-appearance: none;
+}
+.table-responsive::-webkit-scrollbar:horizontal {
+    height: 11px;
+}
+.table-responsive::-webkit-scrollbar-thumb {
+    border-radius: 8px;
+    border: 2px solid white;
+    background-color: rgba(0, 0, 0, .5);
+}
+.table-responsive::-webkit-scrollbar-track { 
+    background-color: #fff; 
+    border-radius: 8px; 
+} 
```

Comment:
Couple things here.

1) This seems to work on Chrome/Safari on Mac OS. It looks like we may need to modify the `scrollbar-color` and `scrollbar-width` if we want the styling to be consistent on Firefox. See https://css-tricks.com/the-current-state-of-styling-scrollbars-in-css/ I think you have to change a setting on Mac OS to get the scrollbar to always show in Firefox however.

2) Typically we try to open one PR for each issue. Could you possibly move this to a separate branch and open a separate PR?

Reply:
I looked into this and added some styles for firefox. But I've been testing and it doesn't 100% seem to work so I'm not sure.

I think the Firefox usage on mobile is going to be very low and on a desktop this really isn't a problem. So I'm inclined to leave it as it is. It seems to look good on mobile Chrome.